pandas/core/base.py

```diff
@@ -15,6 +16,24 @@
                             duplicated='IndexOpsMixin')
 
 
+def _validate_sorter(obj, sorter):
+    """
+    Checks whether or not the implementation of 'searchsorted'
+    for a given class instance 'obj' should accept a non-None
+    'sorter' argument, if one is provided. Raises a
+    NotImplementedError if that is not the case.
+    """
```

Comment:
of course not. you need to put the actual impl here as well. The point of these cleanups is to have less, more concise, more easily understandable code. Not more functions which are scattered.